packages/button/CHANGELOG.md

```diff
@@ -1,834 +1,435 @@
 # Change Log
 
+## 2.2.0
```

Comment:
@HeartSquared how do I revert this file to `main` lol

Reply:
https://stackoverflow.com/questions/37972753/git-reset-single-file-in-feature-branch-to-be-the-same-as-in-master

This? I wonder if you could reset hard on the single file 🤔 

Alt, you barely have any changes, so you could always just soft reset to main, then only commit the changes you want :)

Reply:
Yeah this worked:
```git checkout origin/master packages/button/CHANGELOG.md```